src/types.ts

```diff
@@ -97,13 +97,12 @@ export interface BasicStateVariableOptions {
 export interface MappingStateVariableOptions {
   setFunction: {
     functionName: string;
-    keyType: string;
+    keyTypes: string[];
     valueType: string;
-    mappingName: string;
```

Comment:
`mappingName` property appeared to be redundant with `functionName`, which was the one preserved due to cohesion with the other `VariableOptions` types.